contrib/syntax/textmate/Docker.tmbundle/Syntaxes/Dockerfile.tmLanguage

```diff
@@ -25,7 +25,7 @@
 				</dict>
 			</dict>
 			<key>match</key>
-			<string>^\s*\b(FROM)\b.*?\b(AS)\b</string>
+			<string>^\s*\b(?i:(FROM))\b.*?\b(?i:(AS))\b</string>
```

Comment:
I wonder if a [`/i` for the whole expression](https://github.com/kkos/oniguruma/blob/master/doc/RE#L458-L459) would also work;

>  Ignore case option is effective to numbered character. ex. `/\x61/i =~ "A"`


something like
```xml
<string>/^\s*\b(FROM)\b.*?\b(AS)\b/i</string>
```